ui/component/autoplayCountdown/view.jsx

```diff
@@ -155,7 +160,11 @@ function AutoplayCountdown(props: Props) {
               iconRight={skipPaid ? ICONS.WALLET : ICONS.REPLAY}
               onClick={() => {
                 setTimerCanceled(true);
-                doReplay();
+                if (skipPaid) {
+                  doOpenModal(MODALS.AFFIRM_PURCHASE, { uri, cancelCb: () => setTimerCanceled(false) });
```

Comment:
- [ ] "Play previous" not vertically centered
- [ ] Wallet icon would be cleaner if on the left as well.

![image](https://user-images.githubusercontent.com/64950861/176471240-6ba0e4e0-df73-47a1-af9f-88edaf780f43.png)